frontend/test/metabase/scenarios/collections/personal-collections.cy.spec.js

```diff
@@ -0,0 +1,114 @@
+import { restore, popover, modal } from "__support__/cypress";
+import { USERS } from "__support__/cypress_data";
+
+describe("personal collections", () => {
+  beforeEach(() => {
+    restore();
+    cy.server();
+  });
+
+  describe("admin", () => {
+    beforeEach(() => {
+      cy.signInAsAdmin();
+      // Turn normal user into another admin
+      cy.request("PUT", "/api/user/2", {
+        is_superuser: true,
+      });
+    });
+
+    it("should be able to view their own as well as other users' personal collections (including other admins)", () => {
+      cy.visit("/collection/root");
+      cy.findByText("Your personal collection");
+      cy.findByText("Other users' personal collections").click();
+      cy.location("pathname").should("eq", "/collection/users");
+      cy.findByText(/All personal collections/i);
+      Object.values(USERS).forEach(user => {
+        const FULL_NAME = `${user.first_name} ${user.last_name}`;
+        cy.findByText(FULL_NAME);
+      });
+    });
+
+    it("shouldn't be able to change permission levels on or edit personal collections", () => {
+      cy.visit("/collection/root");
+      cy.findByText("Your personal collection").click();
+      cy.icon("lock").should("not.exist");
+      cy.icon("pencil").should("not.exist");
+      // Visit random user's personal collection
+      cy.visit("/collection/5");
```

Comment:
Are you sure this is a PC and not just a regular collection? Or if it is even an existing collection?

Reply:
It is an existing collection and it's NOSQL's personal collection. But  if we start changing the default snapshot this might change as well. When that happens, I'll have to make some changes around this, probably.